packages/dx-react-scheduler/src/plugins/drag-drop-provider.tsx

```diff
@@ -346,13 +348,28 @@ class DragDropProviderBase extends React.PureComponent<
         </Template>
 
         <Template name="allDayPanel">
-          <TemplatePlaceholder />
-          {renderAppointmentItems(this.allDayDraftAppointments, Container, draftData)}
+          <TemplateConnector>
+            {({ currentView, groupOrientation }) => (
+              <>
+                <TemplatePlaceholder />
+                {groupOrientation?.(currentView.name) !== VERTICAL_GROUP_ORIENTATION
+                  && renderAppointmentItems(this.allDayDraftAppointments, Container, draftData)}
```

Comment:
I think it would be more clear if we use `if () {..} else return null;` approach